powerauth-backend-tests/src/test/java/com/wultra/security/powerauth/test/v31/PowerAuthIdentityVerificationTest.java

```diff
@@ -663,18 +626,12 @@ void largeUploadTest() throws Exception {
                 DocumentMetadataResponseDto doc1 = response.getDocuments().get(0);
                 documentId = doc1.getId();
                 assertNotNull(documentId);
-                if (config.isVerificationOnSubmitEnabled()) {
-                    assertEquals(DocumentStatus.UPLOAD_IN_PROGRESS, doc1.getStatus());
-                } else {
-                    assertEquals(DocumentStatus.VERIFICATION_PENDING, doc1.getStatus());
-                    documentVerificationPending = true;
-                }
+                // TODO - check that this state is correct, I would expect that submitted document is in VERIFICATION_PENDING
```

Comment:
Indeed, it looks strange, but we may investigate `largeUploadTest` later, since this functionality is currently not used.